test/integration/json.jl

```diff
@@ -19,9 +19,19 @@
         @test json_response isa JSON3.Object
         @test json_response == JSON3.read(raw_response)
 
-        dict_response = FHIRClient.request(Dict, client, "GET", patient_request)
+        test_logger = Test.TestLogger(; min_level = Logging.Debug)
```

Comment:
Just to make it explicitly clear:

```suggestion
        test_logger = Test.TestLogger(; min_level = Logging.Debug, catch_exceptions = false)
```